shub/image/build.py

```diff
@@ -41,19 +41,22 @@
 @click.option("-S", "--skip-tests", help="skip testing image", is_flag=True)
 @click.option("-n", "--no-cache", is_flag=True,
               help="Do not use cache when building the image")
+@click.option("-b", "--build-arg", multiple=True,
+              help="Allow to pass build arguments to docker client.")
 @click.option("-f", "--file", "filename", default='Dockerfile',
               help="Name of the Dockerfile (Default is 'PATH/Dockerfile')")
-def cli(target, debug, verbose, version, skip_tests, no_cache, filename):
-    build_cmd(target, version, skip_tests, no_cache, filename=filename)
+def cli(target, debug, verbose, version, skip_tests, no_cache, build_arg, filename):
+    build_cmd(target, version, skip_tests, no_cache, build_arg, filename=filename)
 
 
-def build_cmd(target, version, skip_tests, no_cache, filename='Dockerfile'):
+def build_cmd(target, version, skip_tests, no_cache, build_arg, filename='Dockerfile'):
     config = load_shub_config()
     create_scrapinghub_yml_wizard(config, target=target, image=True)
     client = utils.get_docker_client()
     project_dir = utils.get_project_dir()
     image = config.get_image(target)
     image_name = utils.format_image_name(image, version)
+    build_args = dict(a.split('=') for a in build_arg)
```

Comment:
In theory, there might be build args with values having "=" inside, let's split by the first character only?
```suggestion
    build_args = dict(a.split('=', 1) for a in build_arg)
```

Reply:
you are right. I will fix.

Reply:
Done.